src/IssueViz/Editor/LocationTagging/IssueLocationStoreAggregator.cs

```diff
@@ -72,6 +72,19 @@ public IEnumerable<IAnalysisIssueLocationVisualization> GetLocations(string file
             return locationStores.SelectMany(x => x.GetLocations(filePath));
         }
 
+        public void RefreshOnBufferChanged(string affectedFilePath)
```

Comment:
Update the composite to forward all `RefreshOnBufferChanged` calls, same as for the existing `Refresh` implementation.